modules/core/src/main/java/org/apache/ignite/internal/StripedExecutorMXBeanAdapter.java

```diff
@@ -40,7 +40,12 @@
 
     /** {@inheritDoc} */
     @Override public void checkStarvation() {
-        exec.checkStarvation();
+        exec.isStarvationDetected();
+    }
+
+    /** {@inheritDoc} */
+    @Override public boolean isStarvationDetected() {
```

Comment:
I don't like that the name sounds like a simple getter but is actually supposed to do work with the main effect being writing a message to the log. How about renaming this to `detectStarvation`?

Reply:
Aaand please mark checkStarvation @Deprecated so that we can ditch it in 3.0

Reply:
done